docs/configuration-validation.md

```diff
@@ -17,3 +17,17 @@ If you were presented with configuration error (or a warning, depending on `conf
 **Note**: In a warning mode (with `configValidationMode: warn` set in configuration) Framework commands are not blocked in any way, e.g. `sls deploy` will still attempt to deploy the service normally (still depending on the source of the warning, success of a deployment may vary)
 
 If you find this functionality problematic, you may also turn it off with `configValidationMode: off` setting.
+
+## Configuration
+
+### Where
```

Comment:
I would skip this nesting to not overcomplicate things - just keep top-level `Configuration`